### PR TITLE
pkg/arduino_sdi_12: bump version, fix patches

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

Mostly doc changes since the last commit hash, patches now apply without the 3-way merge.

### CODE REVIEW

File: pkg/arduino_sdi_12/patches/0003-RIOT-Fix-for-SAMD-boards.patch [hunk 1]
```diff
@@ -11,32 +11,32 @@ it hangs.
...
    // Set the timer prescalers back to original values
 -  // NOTE:  This does NOT reset SAMD board pre-scalers!
    sdi12timer.resetSDI12TimerPrescale();
  }
  
-@@ -266,7 +265,6 @@ void SDI12::end() {
+@@ -264,7 +263,6 @@ void SDI12::end() {
    setState(SDI12_DISABLED);
-   _activeObject = NULL;
+   _activeObject = nullptr;
```
Comment (jue89) [inline pkg/arduino_sdi_12/patches/0003-RIOT-Fix-for-SAMD-boards.patch:33]: Good catch

**Review by jue89 — APPROVED** [review]
Murdock is happy. So am I.

File: pkg/arduino_sdi_12/patches/0003-RIOT-Fix-for-SAMD-boards.patch (hunk 1 above)
Comment (jue89) [reply pkg/arduino_sdi_12/patches/0003-RIOT-Fix-for-SAMD-boards.patch:33]: Oh, it's just to context :D Diffs in diffs are always weird. Never mind ...

**Comment by jue89** [comment]
I guess ci-prod needs another trigger. Is there a way to do it with retriggering Murdock with another 2 hours CI run?

**Comment by kaspar030** [comment]
> I guess ci-prod needs another trigger. Is there a way to do it with retriggering Murdock with another 2 hours CI run?

disable compilation ...

**Comment by kaspar030** [comment]
(I confirm it was green on ci.riot-os.org)

**Comment by kfessel** [comment]
see #18810 we need a backport of this

**Pull request merged**